packages/backend/src/modules/dashboard/dashboard.repository.ts

```diff
@@ -1,27 +1,49 @@
-import { CreateTaskCommandDTO } from 'shared'
+import { CreateTaskCommandDTO, UpdateTaskCommand } from 'shared'
 import DashboardModel from './Dashboard'
 import { Task } from '../task/task'
 
 export class DashboardRepository {
   private readonly _dashboardModel = DashboardModel
 
+  private findColumnIndex(id: string, array: []): number {
+    const indexNumber = array.findIndex((x: { _id: string }) => String(x._id) === id)
+    return indexNumber
+  }
+
   constructor(boardModel: any) {
     this._dashboardModel = boardModel
   }
 
   async addNewTaskToDashboard(createTaskCommand: CreateTaskCommandDTO, savedTask: Task) {
     const dashboard = await this._dashboardModel.findById(createTaskCommand.idDashboard)
     const idCol = { idColumn: createTaskCommand.idColumn }
-    const columnIndex = dashboard.columns.findIndex(
-      (x: { _id: string }) => String(x._id) === idCol.idColumn
-    )
+    const columnIndex = this.findColumnIndex(idCol.idColumn, dashboard.columns)
 
     dashboard.columns[columnIndex].tasks.push(savedTask)
     await dashboard.save()
     console.log(
       `Dashboard repository: task ${savedTask} added to idDashboard: ("${createTaskCommand.idDashboard}"), idColumn: ("${createTaskCommand.idColumn}")`
     )
   }
+
+  async updateTaskOnDashboard(updateTaskCommand: UpdateTaskCommand, id: string) {
+    const dashboard = await this._dashboardModel.findById(updateTaskCommand.idDashboard)
+    const idCol = { idColumn: updateTaskCommand.idColumn }
```

Comment:
Why make it an object, if it's not used anywhere as an object?

Reply:
Done